pkg/apis/core/types_shoot.go

```diff
@@ -500,6 +500,9 @@ type ClusterAutoscaler struct {
 	// Expander defines the algorithm to use during scale up (default: least-waste).
 	// See: https://github.com/gardener/autoscaler/blob/machine-controller-manager-provider/cluster-autoscaler/FAQ.md#what-are-expanders.
 	Expander *ExpanderMode
+	// ExpanderConfig defines the configuration of the CA expander.
```

Comment:
Nit: This doc string isn't quite helpful - what is the CA expander/what is it used for?